### ISSUE TITLE
Reviving and Connecting bodyparts using "Blood vessel wire"

### ISSUE THREAD

**Issue by TamasB14** [studio-minus/ppg-bugs-and-requests#542]
**Feature description**
When I amputate someone's bodypart and I want to sew back that bodypart back to the body the with **Blood Vessel Wire** should be able to revive that old and amputated bodypart and let it function normally. If a leg gets torn off, the human can no longer use it but if I sew it back using the blood vessel wire, it'll be able to *somewhat* stand or atleast stumble a bit.

**Feature purpose**
Adding dynamic amputation and transplantation system of bodyparts. Making medical items alot more dynamic. 

**Additional context**
Resewing spine with the blood vessel wire will still paralyse downer body, but instead of showing "X" on the bodyparts, it will be alive.
(Unless using some sort of " fixed bone marrow wire" which might be my next feature request.)

**Comment by Cboyjet4** [comment]
https://github.com/studio-minus/ppg-bugs-and-requests/issues/230 and https://github.com/studio-minus/ppg-bugs-and-requests/issues/412 (somewhat)

**Comment by fhgb54643689** [comment]
this is a great request

**Comment by ToxicSnake48** [comment]
I would love this

**Comment by TamasB14** [comment]
> #230 and #412 (somewhat)

Yeah that #230 one is pretty close to my suggestion but not entirely the same with some minor tweaks.

**Comment by Cboyjet4** [comment]
this is still a duplicate

> > #230 and #412 (somewhat)
> 
> Yeah that #230 one is pretty close to my suggestion but not entirely the same with some minor tweaks.
